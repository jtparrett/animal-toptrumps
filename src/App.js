import React from "react";
import { ThemeProvider, theme, CSSReset } from "@chakra-ui/core";
import { ApolloProvider } from "@apollo/client";

import { client } from "./ApolloClient";
import { Catalogue } from "./pages";

export const App = () => (
  <ThemeProvider theme={theme}>
    <CSSReset />
    <ApolloProvider client={client}>
      <Catalogue />
    </ApolloProvider>
  </ThemeProvider>
);
