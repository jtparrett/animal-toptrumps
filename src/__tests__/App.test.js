import React from "react";
import { render } from "@testing-library/react";
import { App } from "../App";

test("renders", () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
