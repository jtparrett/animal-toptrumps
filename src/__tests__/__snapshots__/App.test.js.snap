// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="css-f5mycz"
  >
    <div
      class="css-1uanfen"
    >
      <h1
        class="css-4kl02x"
      >
        Catalogue
      </h1>
      <button
        class="css-c4fa1y"
        type="button"
      >
        Create animal
        <svg
          aria-hidden="true"
          class="css-yyruks"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M0,12a1.5,1.5,0,0,0,1.5,1.5h8.75a.25.25,0,0,1,.25.25V22.5a1.5,1.5,0,0,0,3,0V13.75a.25.25,0,0,1,.25-.25H22.5a1.5,1.5,0,0,0,0-3H13.75a.25.25,0,0,1-.25-.25V1.5a1.5,1.5,0,0,0-3,0v8.75a.25.25,0,0,1-.25.25H1.5A1.5,1.5,0,0,0,0,12Z"
            fill="currentColor"
          />
        </svg>
      </button>
    </div>
    <div
      class="css-2zdq89"
    >
      <article
        class="css-19xshe8"
      >
        <div
          class="css-1lm5fr4"
        >
          <div
            class="css-14nbnbh-Skeleton"
          />
          <div
            class="css-li67zv-Skeleton"
          >
            <div
              class="css-emr9vq"
            >
              <p
                class="css-12kwuzl"
              >
                Type
              </p>
              <p
                class="css-1t24val"
              />
            </div>
          </div>
          <div
            class="css-li67zv-Skeleton"
          >
            <div
              class="css-emr9vq"
            >
              <p
                class="css-12kwuzl"
              >
                Diet type
              </p>
              <p
                class="css-1t24val"
              />
            </div>
          </div>
          <div
            class="css-li67zv-Skeleton"
          >
            <div
              class="css-emr9vq"
            >
              <p
                class="css-12kwuzl"
              >
                Extinct
              </p>
              <p
                class="css-1t24val"
              >
                Not extinct
              </p>
            </div>
          </div>
        </div>
      </article>
      <article
        class="css-19xshe8"
      >
        <div
          class="css-1lm5fr4"
        >
          <div
            class="css-14nbnbh-Skeleton"
          />
          <div
            class="css-li67zv-Skeleton"
          >
            <div
              class="css-emr9vq"
            >
              <p
                class="css-12kwuzl"
              >
                Type
              </p>
              <p
                class="css-1t24val"
              />
            </div>
          </div>
          <div
            class="css-li67zv-Skeleton"
          >
            <div
              class="css-emr9vq"
            >
              <p
                class="css-12kwuzl"
              >
                Diet type
              </p>
              <p
                class="css-1t24val"
              />
            </div>
          </div>
          <div
            class="css-li67zv-Skeleton"
          >
            <div
              class="css-emr9vq"
            >
              <p
                class="css-12kwuzl"
              >
                Extinct
              </p>
              <p
                class="css-1t24val"
              >
                Not extinct
              </p>
            </div>
          </div>
        </div>
      </article>
      <article
        class="css-19xshe8"
      >
        <div
          class="css-1lm5fr4"
        >
          <div
            class="css-14nbnbh-Skeleton"
          />
          <div
            class="css-li67zv-Skeleton"
          >
            <div
              class="css-emr9vq"
            >
              <p
                class="css-12kwuzl"
              >
                Type
              </p>
              <p
                class="css-1t24val"
              />
            </div>
          </div>
          <div
            class="css-li67zv-Skeleton"
          >
            <div
              class="css-emr9vq"
            >
              <p
                class="css-12kwuzl"
              >
                Diet type
              </p>
              <p
                class="css-1t24val"
              />
            </div>
          </div>
          <div
            class="css-li67zv-Skeleton"
          >
            <div
              class="css-emr9vq"
            >
              <p
                class="css-12kwuzl"
              >
                Extinct
              </p>
              <p
                class="css-1t24val"
              >
                Not extinct
              </p>
            </div>
          </div>
        </div>
      </article>
    </div>
  </div>
</DocumentFragment>
`;
